cnm.go

```diff
@@ -136,23 +137,37 @@ func (n *cnm) createEndpointsFromScan(networkNSPath string) ([]Endpoint, error)
 
 		if netIface.iface.Name == "lo" {
 			continue
-		} else {
-			endpoint, err = createNetworkEndpoint(idx, uniqueID, netIface.iface.Name)
-			if err != nil {
-				return []Endpoint{}, err
+		}
+
+		err := doNetNS(networkNSPath, func(_ ns.NetNS) error {
+			// Check if interface is a physical interface. Do not create
+			// tap interface/bridge if it is.
+			isPhysical := checkPhysicalIface(netIface.iface.Name)
```

Comment:
Might be useful to log `isPhysical` along with the ifname.

Reply:
Yes, will be useful to log that as well. Will add it.